### DISCUSSION TITLE
How can I use conditional Components in Trame?

### DISCUSSION THREAD

**Discussion by dougbrennan** [Kitware/trame#506] [General] [answered]
I'm trying to use conditional Components in Trame. Specifially, I want to conditionally display VtkRemoteLocalView, however this doesn't support the v_if property so I figured I would just create two Components, one with and one without the VtkRemoteLocalView and then use the component :is directive to choose which one to display.

But - it seems like the ":is" directive won't work in Python - likely becaise "is" is a reserved word. So when I try to add something like 
`vuetify.Compoent (is=("active_component",))`
I get an error.
Does anyone know how to do this? Are there any examples maybe?
I'm not entirely sure I got the syntax for my Component definitions right either..... but one step at time. 

Thanks in advance for any guidance!!!

**Comment by jourdain** [comment, marked as answer]
I'm not sure what you mean by not supporting `v_if`. Since it is a directive it is supported by every component by default. 

You can use `raw_attrs=[':is="active_component"'],`  ([see api](https://trame.readthedocs.io/en/latest/core.widget.html#trame_client.widgets.core.AbstractElement))

**Reply by dougbrennan** [reply]
Well that's embarrasing.......  I got fooled by a combination of misleading Intellisens and putting my keyword arge before my poistional args.....
Thanks Sebastien - that makes life much simpler
